### ISSUE TITLE
@EuropeControllerLogin: AuthCode was not found

### ISSUE THREAD

**Issue by rolf-thomas** [Hacksore/bluelinky#87]
Anybody else experiencing a "@EuropeControllerLogin: AuthCode was not found" error when trying to poll a EU EV on the Hyundai servers? The script was working perfectly before. Stoped working a couple of days ago. Bluelinky version 5.2.3.

Testing with the debug.js script + config.json confirms the problem:
````
$ node ./debug.js
? What Region are you in? EU
{ region: 'EU' }
Logging in...
(node:8797) UnhandledPromiseRejectionWarning: Error [ERR_UNHANDLED_ERROR]: Unhandled error. ('@EuropeControllerLogin: AuthCode was not found')
    at t.emit (events.js:304:17)
    at t.<anonymous> (/Users/x/Code/HyundaiKona/bluelinky/dist/index.js:2:41136)
    at /Users/x/Code/HyundaiKona/bluelinky/dist/index.js:2:1955
    at Object.throw (/Users/x/Code/HyundaiKona/bluelinky/dist/index.js:2:2060)
    at a (/Users/x/Code/HyundaiKona/bluelinky/dist/index.js:2:850)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:8797) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:8797) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
````

**Comment by PierreLevres** [comment]
Login with the app. You’ll see you have to change your password or pin.

**Comment by rolf-thomas** [comment]
Thank you for reaching out, Pierre. That worked. 

The Bluelink app was working fine without any changes (but might have been only a matter of time). So I changed the pin in the app – doesn't fix the issue. Changed the password in app – solves the problem (same password by the way). Was then kind of pressed to create a Hyundai account and link the Bluelink account to it. Did just that. I have a feeling this might have been the root of the problem on the Hyundai server side? Maybe?

![hyundai-account-nagscreen](https://user-images.githubusercontent.com/9148913/103170435-7dc57280-4844-11eb-95c0-260fff44948a.jpg)
